File: itcloud/src/main/java/com/itinfo/itcloud/repository/HostInterfaceSampleHistoryRepository.java
package com.itinfo.itcloud.repository;

import com.itinfo.itcloud.repository.entity.HostInterfaceSamplesHistoryEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface HostInterfaceSampleHistoryRepository extends JpaRepository<HostInterfaceSamplesHistoryEntity, Integer> {


    // 호스트 - 네트워크
    HostInterfaceSamplesHistoryEntity findFirstByHostInterfaceIdOrderByHistoryDatetimeDesc(UUID hostInterfaceId);
}

File: itcloud/src/main/java/com/itinfo/itcloud/repository/HostInterfaceSampleHistoryRepository.kt
package com.itinfo.itcloud.repository

import com.itinfo.itcloud.repository.entity.HostInterfaceSamplesHistoryEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface HostInterfaceSampleHistoryRepository : JpaRepository<HostInterfaceSamplesHistoryEntity, Int> {
	// 호스트 - 네트워크
	fun findFirstByHostInterfaceIdOrderByHistoryDatetimeDesc(hostInterfaceId: UUID): HostInterfaceSamplesHistoryEntity
}